Lib/asyncio/proactor_events.py

```diff
@@ -107,6 +110,8 @@ def _fatal_error(self, exc, message='Fatal error on pipe transport'):
             self._force_close(exc)
 
     def _force_close(self, exc):
+        if self._empty_waiter is not None:
+            self._empty_waiter.set_exception(exc)
```

Comment:
`exc` can be `None`, for instance when `transport.abort()` is called.

Reply:
Good catch